app/services/submission_service.rb

```diff
@@ -8,22 +8,26 @@ class SubmissionError < StandardError
   end
 
   class << self
-    def create_submission(submission_params, gravity_user_id)
+    def create_submission(submission_params, gravity_user_id, is_convection)
```

Comment:
You can set default value, and don't pass false elsewhere
```suggestion
    def create_submission(submission_params, gravity_user_id, is_convection = false)
```

Reply:
when I set it as `is_convection = false`, the CI complains: `Use keyword arguments when defining method with boolean argument`. I guess it wants me to use `is_convection: false`, but that behaves differently.  

do you know how to overcome this?

Reply:
here: https://app.circleci.com/pipelines/github/artsy/convection/3152/workflows/5201dd79-e92e-4ec5-a2a3-20d8de795de4/jobs/3787

Reply:
Oh, rubocop 
u have 3 options: 
1) disable OptionalBooleanParameter 
2) Use `is_convection: false`, but call the method with the third parameters explicitly specifying the third parameter, such as `SubmissionService.create_submission (params, 'userid', is_convection: true)` 
3) Change it to nil `create_submission(submission_params, gravity_user_id, is_convection = nil)`